src/main/java/com/google/cloud/teleport/bigtable/BigtableToAvro.java

```diff
@@ -106,6 +113,11 @@ public static PipelineResult run(Options options) {
             .withInstanceId(options.getBigtableInstanceId())
             .withTableId(options.getBigtableTableId());
 
+    if (options.getCellsPerColumn().isAccessible()) {
```

Comment:
This would not actually work in a template as the conditional statement is evaluated at compile time (when the template is created) and not at runtime. As a result, the template will either be with or without this option depending on the value of --cellsPerColumn at compile time.

You should be able to verify it in your project.

Reply:
Thanks! That makes sense.

It seems like this feature would be non-trivial to add as part of the template.